src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -17,12 +17,43 @@
     <PackageReadmeFilePath>$(MSBuildThisFileDirectory)../src/PACKAGE.md</PackageReadmeFilePath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageSupportedRID Include="android-arm" />
+    <PackageSupportedRID Include="android-arm64" />
+    <PackageSupportedRID Include="android-x64" />
+    <PackageSupportedRID Include="android-x86" />
+    <PackageSupportedRID Include="linux-arm" />
+    <PackageSupportedRID Include="linux-arm64" />
+    <PackageSupportedRID Include="linux-bionic-arm64" />
+    <PackageSupportedRID Include="linux-bionic-x64" />
+    <PackageSupportedRID Include="linux-musl-arm" />
+    <PackageSupportedRID Include="linux-musl-arm64" />
+    <PackageSupportedRID Include="linux-musl-x64" />
+    <PackageSupportedRID Include="linux-x64" />
+    <PackageSupportedRID Include="maccatalyst-arm64" />
+    <PackageSupportedRID Include="maccatalyst-x64" />
+    <PackageSupportedRID Include="osx-arm64" />
+    <PackageSupportedRID Include="osx-x64" />
```

Comment:
I would like @ericstj to comment on this PR (as I'm currently out). IMO it would be better to not rely on a static list here as that requires more things to touch when adding a new runtime RID package.

From a given RID it should be possible to get the parent RIDs either via an SDK task or via a custom msbuild task.

Reply:
We'd need the other way round, going from parent RID to child RIDs since we need to include those in the parent-RID-package. I'm not sure that will be easier. I can try basing it off of the `runtime.<RID>.runtime.native.System.IO.Ports.proj` files though.

Reply:
pushed an update